ImageInLib/ImageInLib/src/segmentation3D_common.c

```diff
@@ -0,0 +1,30 @@
+#include "segmentation3D_common.h"
+#include <math.h>
+
+dataType approx_U_pq(dataType U_p, dataType U_q, dataType A_pq)
+{
+	if (A_pq > 0)
+	{
+		return U_p;
+	}
+	else if (A_pq < 0)
+	{
+		return U_q;
+	}
```

Comment:
What about a remaing case? What if A_pq == 0?

Reply:
This was mistake, should have added this